srv/salt/_modules/cephimages.py

```diff
@@ -6,20 +6,31 @@
 from glob import glob
 from subprocess import Popen, PIPE
 
+def _is_installed(name):
+    try:
+        devnull = open(os.devnull)
+        cmd = "type ".format(name)
```

Comment:
Further down we are constructing the cmd like this: `cmd = [ '/usr/bin/rbd', '-p', pool, 'ls' ]`

If I'm reading this correctly, here it will be something like `['type rados']` - would `['type', name]` be better?